server/service/src/sync/translations/master_list_line.rs

```diff
@@ -71,7 +82,7 @@ mod tests {
         let translator = MasterListLineTranslation {};
 
         let (_, connection, _, _) =
-            setup_all("test_master_list_line_translation", MockDataInserts::none()).await;
+            setup_all("test_master_list_line_translation", MockDataInserts::all()).await;
```

Comment:
With new none check... the master list line will always be ignored cause there are no master lists in the mock database...

Reply:
Could you please add a comment on why all() is needed? i.e. that test_data::test_pull_upsert_records() requires a master_list from the mock data.

Could you also please add a test for the new `Ignored` case?